docs/databases/swift-kuery.html

```diff
@@ -92,9 +99,10 @@ <h2 class="heading-2"><span class="blue-text">Step 2c:</span> Create connection
           <p>Add SwiftKuery and SwiftKueryPostgreSQL to your package.swift</p>
           <p>Inside `KueryRoutes.swift`, import SwiftKuery and SwiftKueryPostgreSQL:</p>
           <pre><code class="language-swift">import SwiftKuery
-  import SwiftKueryPostgreSQL</code></pre>
-          <p>Beneath your import statements, create a connection pool of connections:</p>
-          <pre><code class="language-swift">let pool = PostgreSQLConnection.createPool(host: "localhost", port: 5432, options: [.databaseName("bookstoredb")], poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50))</code></pre>
+import SwiftKueryPostgreSQL</code></pre>
+          <p>Inside the `App` extension, create a connection pool of connections:</p>
+          <pre><code class="language-swift">static let pool = PostgreSQLConnection.createPool(host: "localhost", port: 5432, options: [.databaseName("bookstoredb")], poolOptions: ConnectionPoolOptions(initialCapacity: 10, maxCapacity: 50))
+// Create table instance here</code></pre>
```

Comment:
There are two references below to Postgres which should be MySQL:
"On macOS we can use Homebrew to install Postgres:" and "On Linux we can use `apt-get` to install Postgres:"